usr/local/bin/apt

```diff
@@ -13,6 +13,7 @@ def usage():
     print("Commands:")
     print("  add-repository   - Add entries to apt sources.list")
     print("  autoclean        - Erase old downloaded archive files")
+    print("  autopurge        - Purge packages with automatically all unused packages")
     print("  autoremove       - Remove automatically all unused packages")
```

Comment:
Maybe the same respective change here also just to be consistent.

Reply:
How is that? "Remove packages with their configuration files and automatically remove all unused packages"